tabs/src/types/global.d.ts

```diff
@@ -68,3 +68,20 @@ declare module '*.png' {
   const content: any;
   export default content;
 }
+
```

Comment:
Nope - but these back into the local file. These don't need to be global. I moved them there for a reason :-)